### PR TITLE
EX-1514 - Style the sicherearbeit.at article component

### PR DESCRIPTION
**Pull request by Bejor**
How to test:
Open Article Demo Page

Screenshot 2021-09-15 at 17-02-48 Corona-Pandemie trifft Frauen härter
Screenshot 2021-09-15 at 17-03-07 Corona-Pandemie trifft Frauen härter

### CODE REVIEW

File: src/components/ex-article/ex-article.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    @screen md {
+      @apply p-0 m-0;
+    }
+
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      @apply text-primary pt-4 font-bold leading-8;
```
Comment (martinkastler) [inline src/components/ex-article/ex-article.scss:14]: As general remark for this and all future websites: The headline must visually always be closer to the following paragraph than the preceding paragraph.

This is just an example without definitive pixel values. It illustrate the idea:
image


image

image

**Review by martinkastler — CHANGES_REQUESTED** [review]

**Comment by martinkastler** [comment]
I found another issue: The author and date look fine with Max Mustermann. But names can and will be longer. I suggest on mobile the date is always pushed into the next row and the border-right can be removed.

File: src/components/ex-article/ex-article.scss (hunk 1 above)
Comment (Bejor) [reply src/components/ex-article/ex-article.scss]: Changed spacing also adding link highlighting in text

**Comment by Bejor** [comment]
> I found another issue: The author and date look fine with Max Mustermann. But names can and will be longer. I suggest on mobile the date is always pushed into the next row and the border-right can be removed.

Changed this

**Review by martinkastler — APPROVED** [review]

**Pull request merged**